modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/deployer/ObjectDefinitionDeployerImpl.java

```diff
@@ -52,7 +52,7 @@
 public class ObjectDefinitionDeployerImpl implements ObjectDefinitionDeployer {
 
 	@Override
-	public List<ServiceRegistration<?>> deploy(
+	public synchronized List<ServiceRegistration<?>> deploy(
```

Comment:
we could make synchronized code blocks that would be slightly more performant but I think it's Ok.